docs/API.md

```diff
@@ -1551,6 +1551,73 @@ Example: `GET ~bob/trusty/wordpress-42/meta/id-series`
 }
 ```
 
+#### GET *id*/meta/common-extra-info
+
+The meta/common-extra-info path reports any common metadata recorded for 
+the base entity. 
```

Comment:
can you format this paragraph so that the next sentence carries on from this one?